skills/enso/__init__.py

```diff
@@ -12,6 +13,10 @@ def get_enso_skill(
 
     if name == "get_tokens":
         return EnsoGetTokens(api_token=api_token, main_tokens=main_tokens, store=store, agent_id=agent_id)
+    if name == "get_wallet_approvals":
```

Comment:
Please maintain consistency in the config name, class name, and tool name. The config name can omit the enso prefix, but try to keep the rest consistent.